packages/editor/src/client/panel/imageEditPanel.tsx

```diff
@@ -394,3 +398,9 @@ export function ImagedEditPanel({id, file, onClose, onSave, imageMetaData}: Imag
     </>
   )
 }
+
+const CheckedPermissionComponent = createCheckedPermissionComponent(
```

Comment:
Could you outline the purpose of this createCheckedPermissionComponent? Please add it to the docs.